examples/hooks-next/.eslintrc

```diff
@@ -0,0 +1,6 @@
+{
+  "rules": {
```

Comment:
Building and linting fail if ESLint doesn't load the Next.js plugin.

```suggestion
  "extends": ["next"],
  "rules": {
```

Reply:
it's just a warning right? otherwise the example test doesn't work

Reply:
On top of the warning, it fails when it's run outside of the react-instantsearch repository structure:

```sh
curl https://codeload.github.com/algolia/react-instantsearch/tar.gz/feat/next-hooks-example | tar -xz --strip=2 react-instantsearch-feat-next-hooks-example/examples/hooks-next
```

![image](https://user-images.githubusercontent.com/154633/148236879-a6ed1c72-9a22-40c0-8909-79762c5e06b9.png)

I'm not sure why, maybe ESLint tries to find missing configuration in parent directories and since we have one at the root of **react-instantsearch** its okay.

Reply:
I think it's fixed now